### ISSUE TITLE
Fail to upload factory and default firmware

### ISSUE THREAD

**Issue by JpEncausse** [Xinyuan-LilyGO/T-Dongle-S3#10]
Hello,
- I just received the device then plugged-in it boot and works
- I tried to compile/upload with PlatformIO the example/firmware is fail while uploading, the display is black
- I tried to upload the default firmware with esptool it fail at 50/60% (screenshots)
- I tried to change the driver (some issue on intenet) didn't change anything
- I have other TTGO Display, etc ... all other works, compile and upload.

![error](https://github.com/Xinyuan-LilyGO/T-Dongle-S3/assets/1140867/5df1ab0b-c594-49f2-bcc9-a151ed4dfef9)

I don't know what to do,
1. I need to upload the firmware with esptool to validate it works
2. then uderstand why I can't upload from PlatformIO

I switch port, reduce the bandwith, etc ... nothing change
Now when I monitor the device from VSCode I have a loop :

```
Build:Mar 27 2021
rst:0x3 (RTC_SW_SYS_RST),boot:0x8 (SPI_FAST_FLASH_BOOT)
Saved PC:0x403bb22e
SPIWP:0xee
mode:DIO, clock div:1
load:0x3fcd0108,len:0x43c
load:0x403b6000,len:0xbd0
load:0x403ba000,len:0x29c8
SHA-256 comparison failed:
Calculated: 27beaa143e4753b6cdeed5b388d3ebbf4c921ce01156fd10673139a323a25557
Expected: 6eb14ec480aff4f05e39378c41406102ac163573741e63b6cfd73509ef44e3bd
Attempting to boot anyway...
entry 0x403b61d8
ESP-ROM:esp32s3-20210327
```

**Comment by JpEncausse** [comment]
- I tried to restart : nothing change
- I wait and retry : the firmware as been updated
- I try example project : it is VERY random, sometimes it upload up to 8% sometimes it works sometimes it's 80% or 60%

It's like the device get disconnected while uploading or something like that. I tried multiple USB port same issue.

**Comment by JpEncausse** [comment]
So pressing the button while inserting the USB, seems to always help uploading the firmware with esptool.exe

But otherwise the upload of the firmware or the from platformio fail most of the time.
I assume the USB key do something ?? that break upload ?

**Comment by JpEncausse** [comment]
> 1.Pressed BOOT Button 2. Insert USB 3. Upload
Yes that's the workflow I use to save the dongle with master firmware when nothing works

But how do you handle regular developpement in PlatformIO ? Do you follow these steps each time before pressing Upload of your code ?!

I don't have these issue with other LILYGO devices (camera, display, etc ... works fine) Is it a specific issue to this type of device ? or my device has flaws?

**Comment by lewisxhe** [comment]
No, the program does not need to operate in this way during normal operation. It is only necessary to operate in this way when the wrong program is running causing chip abnormalities.

**Comment by JpEncausse** [comment]
There is still a lot of difficulities to upload custom code with PlatformIO to the device in normal mode. 
It fail > 50% of the time and need to reset to default firmware (plug / push button) before trying again
Is it normal behavior ?

**Comment by lewisxhe** [comment]
Try reducing the Baud by 115200. I have no problem with the test here.
